e2e/tests-riff/apis.bats

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bats
```

Comment:
There's probably a better way than creating separate riff tests, but we can fix that separately